### PR TITLE
docs: add how to create a minimal repo in bug report template

### PR DESCRIPTION
**Pull request by alan-agius4**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [x] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by filipesilva — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/1-bug-report.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Please create and share minimal reproduction of the issue starting with this tem
 <!-- ✍️--> https://stackblitz.com/...
 
 <!--
-If StackBlitz is not suitable for reproduction of your issue, please create a minimal GitHub repository with the reproduction of the issue. Share the link to the repo below along with step-by-step instructions to reproduce the problem, as well as expected and actual behavior.
+If StackBlitz is not suitable for reproduction of your issue, please create a minimal GitHub repository with the reproduction of the issue. A good way to make a minimal repro is to create a new app via `ng new repro-app` and adding the minimum possible code to show the problem. Share the link to the repo below along with step-by-step instructions to reproduce the problem, as well as expected and actual behavior.
```
Comment (gkalpak) [inline .github/ISSUE_TEMPLATE/1-bug-report.md]: Suggestions:
- _repro_ --> _reproduction_
- _to create [...] and adding_ --> _to create [...] and add_

**Review by gkalpak — APPROVED** [review]
👌

File: .github/ISSUE_TEMPLATE/1-bug-report.md (hunk 1 above)
Comment (ericmartinezr) [reply .github/ISSUE_TEMPLATE/1-bug-report.md]: Note that you can import a repo to stackblitz (https://stackblitz.com/docs under **Importing Projects**).

Comment (gkalpak) [reply .github/ISSUE_TEMPLATE/1-bug-report.md]: @ericmartinezr, can you explain why/how this would be useful in this context?

**Pull request closed without merging**